src/main/java/org/apache/maven/buildcache/xml/CacheConfigImpl.java

```diff
@@ -81,6 +81,7 @@ public class CacheConfigImpl implements org.apache.maven.buildcache.xml.CacheCon
     public static final String CONFIG_PATH_PROPERTY_NAME = "maven.build.cache.configPath";
     public static final String CACHE_ENABLED_PROPERTY_NAME = "maven.build.cache.enabled";
     public static final String CACHE_LOCATION_PROPERTY_NAME = "maven.build.cache.location";
+    public static final String REMOTE_URL_PROPERTY_NAME = "maven.build.cache.remoteUrl";
```

Comment:
Are those changes related to this PR ?

Reply:
Yes, with this change is to set remote cache url from command line and it's currently used in integration tests to test remote cache interactions using WireMock. The change allows to use using dynamic WireMock ports  to prevent port conflicts in tests